### ISSUE TITLE
dialog fails to come up correctly the second time...

### ISSUE THREAD

**Issue by lloyd** [mozilla/persona#589]
```
12:13 < lloyd> yay
12:13 < lloyd> so we have a new 5 star issue in dev
12:13 < lloyd> benadida | stomlinson: log into dev.myfavoritebeer.org, log out of same, log back into same 
12:14 < lloyd> notice at the last step the dialog flickrs and closes automatically
```

**Comment by lloyd** [comment]
The flicker and close behavior is on chrome.  On firefox the dialog just "hangs", in the error console is this message:

```
Error: uncaught exception: complete called for non-existant message: 803165
```

**Comment by benadida** [comment]
that error actually occurs on every call, even the first one.

**Comment by lloyd** [comment]
the bug was introduced in be381af4fd5dfc1e38e4e30ee162c5c4042a45d7

**Comment by lloyd** [comment]
Why is there no closing commit on this bug?

**Comment by benadida** [comment]
@lloyd ? what do you mean? I committed and referenced the issue as closed.

**Comment by bplus-archive** [comment]
Oh.  Teehee.  Github issues for iPhone doesn't give the whole truth.  Sry

**Comment by lloyd** [comment]
yes!  we have regressed on dev.  confirmed and am looking now

**Comment by lloyd** [comment]
resolved in dev, I think by commit 6affd5f4855ee3c40a7423a9df1e4b30c570a9a3
